### ISSUE TITLE
wp_wc_products database table creation error

### ISSUE THREAD

**Issue by nattila1** [woocommerce/woocommerce-product-tables-feature-plugin#135]
wp_wc_products database table didn't created during the installation process. other tables created.
I got the following error. my PHP and SQL version is: 7.2.10-0ubuntu0.18.04.1 / 5.7.24-0ubuntu0.18.04.1
```
[01-Nov-2018 12:18:30 UTC] WordPress adatbázis hiba You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'virtual (virtual),
			  KEY downloadable (downloadable),
			  KEY stock_status (' at line 26 a lekérdezésben 
			CREATE TABLE wp_wc_products (
			  `product_id` bigint(20) NOT NULL,
			  `sku` varchar(100) NULL default '',
			  `image_id` bigint(20) NULL default 0,
			  `height` double NULL default NULL,
			  `width` double NULL default NULL,
			  `length` double NULL default NULL,
			  `weight` double NULL default NULL,
			  `stock_quantity` double NULL default NULL,
			  `type` varchar(100) NULL default 'simple',
			  `virtual` tinyint(1) NULL default 0,
			  `downloadable` tinyint(1) NULL default 0,
			  `tax_class` varchar(100) NULL default '',
			  `tax_status` varchar(100) NULL default 'taxable',
			  `total_sales` double NULL default 0,
			  `price` double NULL default NULL,
			  `regular_price` double NULL default NULL,
			  `sale_price` double NULL default NULL,
			  `date_on_sale_from` datetime NULL default NULL,
			  `date_on_sale_to` datetime NULL default NULL,
			  `average_rating` float NULL default 0,
			  `stock_status` varchar(100) NULL default 'instock',
			  PRIMARY KEY  (`product_id`),
			  KEY image_id (image_id),
			  KEY type (type),
			  KEY virtual (virtual),
			  KEY downloadable (downloadable),
			  KEY stock_status (stock_status)

			) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_hungarian_ci készítette activate_plugin, do_action('activate_woocommerce-product-tables-feature-plugin-1.0-beta-1/woocommerce-product-tables-feature-plugin.php'), WP_Hook->do_action, WP_Hook->apply_filters, wc_custom_product_tables_activate, WC_Product_Tables_Install::activate, dbDelta
```

**Comment by leewillis77** [comment]
This was fixed in e791d36 but I don't think that's in a release yet. If you use the latest master branch it will avoid that error.

**Comment by leewillis77** [comment]
Duplicate of #122
